sfdoc/publish/html.py

```diff
@@ -24,15 +24,17 @@ def __init__(self, htmlpath, rootpath):
         self.htmlpath = htmlpath
         self.rootpath = rootpath
 
-        # meta (URL name, summary, visibility settings)
+        # meta (Python attrname, HTML Name, Required or not)
         for attr, tag_name, optional in (
             ('url_name', 'UrlName', False),
             ('summary', 'description', True),
             ('is_visible_in_csp', 'is-visible-in-csp', False),
             ('is_visible_in_pkb', 'is-visible-in-pkb', False),
             ('is_visible_in_prm', 'is-visible-in-prm', False),
             ('author', settings.ARTICLE_AUTHOR, False),
-            ('docset_id', 'ProductMapUUID', True)  # Should be required but will break a lot of tests,
+            ('docset_id', 'ProductMapUUID', True),  # Should be required but will break a lot of tests,
+            ('Topics__c', "HubTopics", True),
+            ('Article_Type__c', "ArticleType", True),
```

Comment:
Is making these new attributes required going to create any backwards-compatibility problems when publishing existing docs?

Reply:
Sorry: the comment is confusing. I'll fix that. True means optional.